### ISSUE TITLE
spine.model.ajax.js needs to be updated to latest spine

### ISSUE THREAD

**Issue by joelcarrier** [maccman/spine.contacts#4]
I realize it isn't used by the demo, but someone grabbing this demo to use as an example and wanting to convert it to ajax rather than local storage will hit a problem on line 38 of spine.model.ajax.js

**Comment by maccman** [comment]
Yup, I need to get round to it. Going to do all of this in one fell swoop -
and create a new website for Spine too.

On Mon, Aug 15, 2011 at 7:07 PM, joelcarrier <
reply@reply.github.com>wrote:

> I realize it isn't used by the demo, but someone grabbing this demo to use
> as an example and wanting to convert it to ajax rather than local storage
> will hit a problem on line 38 of spine.model.ajax.js
> 
> ## 
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/maccman/spine.contacts/issues/4

## 

Alex MacCaw

+12147175129
@maccman

http://alexmaccaw.co.uk | http://www.leadthinking.com | http://socialmod.com

**Comment by impressiver** [comment]
Just ran into this, it's a quick fix to get it working, but I've been checking out the current source and it looks way more robust. Any idea when there will be a working example for the new stuff?

For anyone else who might run into this: the quick fix is to flip the parameters on the "ajaxSync" method on line 19 (and also the method call on line 52), and you might need to change line 38 to "(record.populate || record.load).call(record, data);" if you want to retain scope.
